test/cucumber/steps/UpdateLattesSteps.groovy

```diff
@@ -0,0 +1,173 @@
+        def article2 = TestAndOperations.buildArticle(title2,journal,issn)
+        researcher = TestAndOperations.addArticleToResearcher(researcher, article1)
+        researcher = TestAndOperations.addArticleToResearcher(researcher, article2)
+        TestAndOperations.createResearcher(researcher)
+        researcher = Researcher.findByCpf(cpf)
+        TestAndOperations.compareResearcherWithCpfAndName(researcher, cpf, name)
+        assert researcher.articles.size() == 2
+        article1 = researcher.articles[0]
+        article2 = researcher.articles[1]
+        assert (TestAndOperations.compareArticle(article1,title1,journal,issn) &&
+                TestAndOperations.compareArticle(article2,title2,journal,issn)) ||
+                (TestAndOperations.compareArticle(article1,title2,journal,issn) &&
+                        TestAndOperations.compareArticle(article2,title1,journal,issn))
+}
+
+And(~/^o arquivo "([^"]*)" tem o Pesquisador "([^"]*)", cpf "([^"]*)" e só tem o artigo "([^"]*)" com journal "([^"]*)" e issn "([^"]*)"$/) {
+    String filename, String name, String cpf, String title, String journal, String issn ->
+        def researcher = TestAndOperations.buildResearcherWithFile(filename)
+        assert TestAndOperations.compareResearcherWithCpfAndName(researcher, cpf, name)
+        assert researcher.articles.size() == 1
+        def article = researcher.articles[0]
+        assert TestAndOperations.compareArticle(article,title,journal,issn)
+}
+
+And(~/^sistema salva um diff no pesquisador  de nome "([^"]*)" e cpf "([^"]*)" informando que o artigo "([^"]*)" foi removido$/) {
+    String name, String cpf, String title ->
+        def researcher = Researcher.findByCpf(cpf)
+        assert TestAndOperations.compareResearcherWithCpfAndName(researcher, cpf, name)
+        assert researcher.diffs.size() == 1
+        assert TestAndOperations.compareDiff(researcher.diffs[0],title,null,2)
+
+}
+
+And(~/^o artigo "([^"]*)" foi removido do sistema$/) {
+    String title ->
+        assert Article.findByTittle(title) == null
+}
+
+And(~/^o pesquisador de cpf "([^"]*)", nome "([^"]*)" e só tem o artigo "([^"]*)" com journal "([^"]*)" e issn "([^"]*)" está cadastrado$/) {
+    String cpf, String name, String title, String journal, String issn ->
+        def researcher = Researcher.findByCpf(cpf)
+        assert TestAndOperations.compareResearcherWithCpfAndName(researcher,cpf,name)
+        assert researcher.articles.size() == 1
+        assert TestAndOperations.compareArticle(researcher.articles[0],title,journal,issn)
+}
+
+Then(~/^Sistema não armazena nenhum novo diff no pesquisador de cpf "([^"]*)"\.$/) {
+    String cpf ->
+        def researcher = Researcher.findByCpf(cpf)
+        assert researcher.diffs.size() == 0
+}
+
+And(~/^sistema salva dois diff no pesquisador  de nome "([^"]*)" e cpf "([^"]*)" informando que o artigo "([^"]*)" foi removido e o artigo "([^"]*)" foi adicionado$/) {
```

Comment:
Esse passo and também não está no cenario correspondente. Ele faz parte do cenario 4.